crates/core_arch/src/x86/sse.rs

```diff
@@ -1470,6 +1470,10 @@ pub unsafe fn _mm_sfence() {
 /// For more info see [`_mm_setcsr`](fn._mm_setcsr.html)
 ///
 /// [Intel's documentation](https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_getcsr)
+#[rustc_deprecated(
+    since = "1.40.0",
+    reason = "modifying MXCSR causes undefined behavior in LLVM, see https://github.com/rust-lang/stdarch/issues/781"
```

Comment:
I do think this should be deprecated but the current reason isn't very convincing on this function since it doesn't modify MXCSR. Perhaps reword it like this:

> accessing floating point exception state causes undefined behavior in LLVM